R/selections.R

```diff
@@ -195,7 +195,7 @@ recipes_eval_select <- function(quos, data, info, ..., allow_rename = FALSE,
   # Return names not positions, as these names are
   # used for both the training and test set and their positions
   # may have changed. If renaming is allowed, add the new names.
-  out <- names(data)[sel]
+  out <- extract_names(data)[sel]
```

Comment:
I think `data` is always a data frame for us, so it is probably fine to simplify this and just use `colnames()` rather than `names()` and avoid adding a new S3 generic